src/Og.php

```diff
@@ -205,6 +212,54 @@ public static function isGroupAudienceField(FieldDefinitionInterface $field_conf
   }
 
   /**
+   * Get the name of the group-audience type field.
+   *
+   * @param $entity_type
+   *   The entity type.
+   * @param $bundle_name
+   *   The bundle name to be checked.
+   * @param $group_type
+   *   Filter list to only include fields referencing a specific group type.
+   * @param $group_bundle
+   *   Filter list to only include fields referencing a specific group bundle.
+   *   Fields that do not specify any bundle restrictions at all are also
+   *   included.
+   *
+   * @return
+   *   Array keyed with the field name and the field label as the value.
+   */
+  public static function getGroupAudienceFields($entity_type, $bundle_name, $group_type = NULL, $group_bundle = NULL) {
```

Comment:
This does return multiple? You van have more than one audience field attached to a bundle.